massa-network-worker/src/node_worker.rs

```diff
@@ -162,14 +120,34 @@ impl NodeWorker {
                     }
                 };
             }
+
+            if let Err(e) = socket_writer.write_half.shutdown().await {
```

Comment:
note: this does not shutdown the reader

Reply:
This line has not been added with the intention to close the reader but to close the writer in a proper way... 

@damip Would you llike me to remove it?

Reply:
oh ok, then:
* if graceful shutdown takes program time, we should remove it and let the kernel handle things in the background 
* if graceful shutdown has no performance impact, we can leave it

Reply:
I think we can just drop abruptly :)